tox.ini

```diff
@@ -63,11 +63,11 @@ deps = pylint == 2.6
 ignore_errors = true
 commands =
          # pylint src
-         pylint src/probnum/diffeq --disable="protected-access,abstract-class-instantiated,too-many-locals,too-few-public-methods,too-many-arguments,unused-argument,missing-module-docstring,missing-function-docstring"
-         pylint src/probnum/filtsmooth --disable="duplicate-code,no-self-use,too-many-locals,too-many-arguments,unused-argument,missing-module-docstring,missing-function-docstring"
-         pylint src/probnum/linalg --disable="attribute-defined-outside-init,too-many-statements,too-many-instance-attributes,too-complex,protected-access,too-many-lines,no-self-use,too-many-locals,redefined-builtin,arguments-differ,abstract-method,too-many-arguments,too-many-branches,duplicate-code,unused-argument,fixme,missing-module-docstring"
-         pylint src/probnum/quad --disable="attribute-defined-outside-init,too-few-public-methods,redefined-builtin,arguments-differ,unused-argument,missing-module-docstring"
-         pylint src/probnum/random_variables --disable="missing-function-docstring"
-         pylint src/probnum/utils --disable="duplicate-code,missing-module-docstring,missing-function-docstring"
-         pylint benchmarks --disable="missing-function-docstring"  # not a work in progress, but final
-         pylint tests --disable="line-too-long,duplicate-code,missing-class-docstring,unnecessary-pass,unused-variable,protected-access,attribute-defined-outside-init,no-self-use,abstract-class-instantiated,too-many-arguments,too-many-instance-attributes,too-many-locals,unused-argument,fixme,missing-module-docstring,missing-function-docstring"
+         pylint src/probnum/diffeq --disable="protected-access,abstract-class-instantiated,too-many-locals,too-few-public-methods,too-many-arguments,unused-argument,missing-module-docstring,missing-function-docstring" --jobs=0
+         pylint src/probnum/filtsmooth --disable="duplicate-code,no-self-use,too-many-locals,too-many-arguments,unused-argument,missing-module-docstring,missing-function-docstring" --jobs=0
+         pylint src/probnum/linalg --disable="attribute-defined-outside-init,too-many-statements,too-many-instance-attributes,too-complex,protected-access,too-many-lines,no-self-use,too-many-locals,redefined-builtin,arguments-differ,abstract-method,too-many-arguments,too-many-branches,duplicate-code,unused-argument,fixme,missing-module-docstring" --jobs=0
+         pylint src/probnum/quad --disable="attribute-defined-outside-init,too-few-public-methods,redefined-builtin,arguments-differ,unused-argument,missing-module-docstring" --jobs=0
+         pylint src/probnum/random_variables --disable="missing-function-docstring" --jobs=0
+         pylint src/probnum/utils --disable="duplicate-code,missing-module-docstring,missing-function-docstring" --jobs=0
+         pylint benchmarks --disable="missing-function-docstring"  # not a work in progress, but final --jobs=0
```

Comment:
maybe add the option before the comment here

Reply:
"maybe" haha 
...of course!